docs/content/preview/yugabyte-voyager/migrate/assess-migration.md

```diff
@@ -139,14 +203,35 @@ You can perform the following steps with these scripts:
 For the most accurate migration assessment, the source database must be actively handling its typical workloads at the time the metadata is gathered. This ensures that the recommendations for sharding strategies and cluster sizing are well-aligned with the database's real-world performance and operational needs.
     {{< /warning >}}
 
-1. View the assessment report.
+1. View the assessment report by configuring the [yugabyted](/preview/reference/configuration/yugabyted/) UI. Use the yugabyted UI to review the assessment report, which includes migration strategies, complexity, and effort estimates.
```

Comment:
These steps confuse me. It looks like it should be
1. Start a cluster
2. Update the config file with cluster details
3. Run the assessemt
4. View the assessment on Migrations tab

"Create a target YugabyteDB cluster as follows" step (currrently step 3) really appears to belong on the individual migration pages in the Prepare the target database section